lib/rubocop/cop/team.rb

```diff
@@ -9,11 +9,16 @@ module Cop
     # For performance reasons, Team will first dispatch cops & forces in two groups,
     # first the ones needed for autocorrection (if any), then the rest
     # (unless autocorrections happened).
-    class Team
+    class Team # rubocop:disable Metrics/ClassLength
```

Comment:
Since this `rubocop:disable` directive is expected to become unnecessary once the warning is resolved in the future, it is preferable to specify it as a block rather than at the end of the line. This prevents cluttering `git blame`. 

```suggestion
    # rubocop:disable Metrics/ClassLength
    class Team
```

Please place the `rubocop:enable` directive after the `end` corresponding to the `class`.